### PR TITLE
fix linter issues

### PR DESCRIPTION
**Pull request by AirinB**

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -30,7 +29,7 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
Comment (annahramova97) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:45]: Good Job!

**Review by annahramova97 — CHANGES_REQUESTED** [review]
Random bug fix is missing

**Comment by AirinB** [comment]
@annahramova97 does it seems good now?

**Review by annahramova97 — APPROVED** [review]
Everything looks fine!

**Comment by AirinB** [comment]
@annahramova97 does it seem ok now?

**Pull request merged**